### ISSUE TITLE
Warnings when building on macOS Catalina

### ISSUE THREAD

**Issue by AlexanderEkdahl** [hajimehoshi/ebiten#984]
```
§ sw_vers -productVersion
10.15.1
§ go build
# github.com/go-gl/glfw/v3.2/glfw
In file included from ../go/pkg/mod/github.com/go-gl/glfw@v0.0.0-20190409004039-e6da0acd62b1/v3.2/glfw/c_glfw_darwin.go:8:
../go/pkg/mod/github.com/go-gl/glfw@v0.0.0-20190409004039-e6da0acd62b1/v3.2/glfw/glfw/src/cocoa_window.m:989:9: warning: multiple methods named 'center' found [-Wobjc-multiple-method-names]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/AppKit.framework/Headers/NSCollectionViewCompositionalLayout.h:601:19: note: using
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/AppKit.framework/Headers/NSWindow.h:312:1: note: also found
```

Inspecting those two files it is not wrong in saying that there are two methods named `center`. Not sure how to proceed. I'd be happy to provide more context if you need anything?

**Comment by hajimehoshi** [comment]
Thanks! I have not tried Catalina yet. I'll try this weekend.

**Comment by AlexanderEkdahl** [comment]
I realize this may be more suited as a ticket for the `glfw` dependency. However, I was under the assumption that this library used metal and would it not be possible to do without the `glfw` dependency?

**Comment by hajimehoshi** [comment]
Ebiten uses glfw's window and input handling so it is not easy to remove the dependency

**Comment by hajimehoshi** [comment]
The message seems a warning. Didn't you see any build results in the directory?

**Comment by AlexanderEkdahl** [comment]
Yes, it seems like a warning but I don't see how it can build without knowing which definition of `center` to use.

I got it building by modifying the `glfw` Objective-C file to be explicit about the type. I see that this change has been incorporated into the 3.3 version of glfw but does not appear to be back ported to 3.2. Not sure how to best proceed.

**Comment by AlexanderEkdahl** [comment]
Fix in glfw: https://github.com/glfw/glfw/commit/bac15f94493938eded7161a7a0ab8827584e3db1

**Comment by hajimehoshi** [comment]
I'm not sure what command you executed, but you should see a binary when building succeeded.

**Comment by AlexanderEkdahl** [comment]
@hajimehoshi My bad. The library builds fine but outputs a warning. I've updated the issue title. This should be resolved if ebiten is ever upgrade to 3.3. Feel free to close this issue.

**Comment by hajimehoshi** [comment]
Thanks!
